.gitignore

```diff
@@ -1,3 +1,5 @@
-/php/tests/node_modules
-/vendor
+/php/tests/node_modules/
+/vendor/
 /phpunit.xml
+/composer.lock
+/composer.phar
```

Comment:
it's your infrastructure, this project does not depend on it - use gitignore_global